README.md

```diff
@@ -32,10 +32,11 @@ contract ExampleContract {
     }
 
     function getBTCUSDPrice(bytes[] memory priceUpdateData) public returns (PythStructs.Price memory) {
-        // Update the prices to be set to the latest values. The `priceUpdateData` data should be
-        // retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
+        // Update the prices to the latest available values and pay the required fee for it. The `priceUpdateData` data
+        // should be retrieved from our off-chain Price Service API using the `pyth-evm-js` package.
```

Comment:
Probably worth linking to the fee docs here https://docs.pyth.network/consume-data/on-demand#fees

Reply:
I think now that the link is below it makes sense. to keep it.